src/rgw/rgw_asio_frontend.cc

```diff
@@ -94,10 +94,9 @@ class AsioConnection : public std::enable_shared_from_this<AsioConnection> {
     RGWRequest req{env.store->get_new_req_id()};
     RGWAsioClientIO real_client{std::move(socket), std::move(request)};
     auto real_client_io = rgw::io::add_reordering(
-                            rgw::io::add_buffering(
-                              rgw::io::add_chunking(
-                                rgw::io::add_conlen_controlling(
-                                  &real_client))));
+                            rgw::io::add_chunking(
```

Comment:
@rzarzynski (NIT) Please keep title of PR <50 characters.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits